docs/tutorial.md

```diff
@@ -0,0 +1,602 @@
+```shell
+juju add-model tutorial
+```
+
+You can now view the model you created above by entering the command `juju status` into the command line. You should see the following:
+```
+Model    Controller  Cloud/Region         Version  SLA          Timestamp
+tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
+
+Model "admin/tutorial" is empty.
+```
+
+
+## Deploy Charmed MySQL
+To deploy Charmed MySQL, all you need to do is run the following command, which will fetch the charm from [Charmhub](https://charmhub.io/mysql?channel=edge) and deploy it to your model:
+```shell
+juju deploy mysql --channel edge
+```
+
+Juju will now fetch Charmed MySQL and begin deploying it to the LXD cloud. This process can take several minutes depending on how provisioned (RAM, CPU, etc) your machine is. You can track the progress by running:
+```shell
+juju status --watch 1s
+```
+
+This command is useful for checking the status of Charmed MySQL and gathering information about the machines hosting Charmed MySQL. Some of the helpful information it displays include IP addresses, ports, state, etc. The command updates the status of Charmed MySQL every second and as the application starts you can watch the status and messages of Charmed MySQL change. Wait until the application is ready - when it is ready, `juju status` will show:
+```
+Model     Controller  Cloud/Region         Version  SLA          Timestamp
+tutorial  overlord    localhost/localhost  2.9.38   unsupported  22:52:47+01:00
+
+App    Version          Status  Scale  Charm  Channel  Rev  Exposed  Message
+mysql  8.0.32-0ubun...  active      1  mysql  edge      95  no       Unit is ready: Mode: RW
+
+Unit      Workload  Agent  Machine  Public address  Ports  Message
+mysql/0*  active    idle   0        10.234.188.135         Unit is ready: Mode: RW
+
+Machine  State    Address         Inst id        Series  AZ  Message
+0        started  10.234.188.135  juju-ff9064-0  jammy       Running
+```
+To exit the screen with `juju status --watch 1s`, enter `Ctrl+c`.
+
+## Access MySQL
+> **!** *Disclaimer: this part of the tutorial accesses MySQL via the `root` user. **Do not** directly interface with the root user in a production environment. In a production environment always create a separate user using [Data Integrator](https://charmhub.io/data-integrator) and connect to MySQL with that user instead. Later in the section covering Relations we will cover how to access MySQL without the root user.*
+
+The first action most users take after installing MySQL is accessing MySQL. The easiest way to do this is via the [MySQL Command-Line Client](https://dev.mysql.com/doc/refman/8.0/en/mysql.html) `mysql`. Connecting to the database requires that you know the values for `host`, `username` and `password`. To retrieve the necessary fields please run Charmed MySQL action `get-password`:
+```shell
+juju run-action mysql/leader get-password --wait
```

Comment:
We are defaulting to root, but how about to keep it explicit?
```suggestion
juju run-action mysql/leader get-password username=root --wait
```

Reply:
Good point, TIL. Applied.